tests/integrationv2/test_client_authentication.py

```diff
@@ -11,116 +11,147 @@
 from utils import invalid_test_parameters, get_parameter_name, get_expected_s2n_version
 
 
+# If we test every available cert, the test takes too long.
+# Choose a good representative subset.
+CERTS_TO_TEST = [
```

Comment:
One of the original goals was to test all the things. Is this not possible? Why is this subset acceptable?

Reply:
I can put it back, but the test was taking more than an hour with the full list for both client and server. This partial list should be a good enough approximation of "all the things" because it's got two RSAs with different sizes, two ECDSAs with different sizes, and a RSA-PSS.

Reply:
https://github.com/awslabs/s2n/issues/2469